components/workflow-mgt/org.wso2.carbon.identity.workflow.mgt.ui/src/main/resources/web/workflow-mgt/template-wf-wizard.jsp

```diff
@@ -239,7 +239,14 @@
 
         var stepOrder = 0;
         function nextWizard(){
-            if(!validateInputs()){
+        <%
+        boolean intTemp = true;
+        if(template !=null && template.getName().equals("External Workflow Template")){
+            intTemp = false;
+        }
+        %>
+            var abc = "<%=intTemp%>";
```

Comment:
Let's try to simplify the logic of using java variables within javascript code

Reply:
Ref - https://github.com/wso2-support/identity-apps/pull/319/files#diff-bffbd72fca2790d1abafde5ff9360318c6859cbcce523989782f98e389150aaeR92

Reply:
will be fix in next PR